src-cr/Components/FoodCard.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+
+class FoodCard extends React.Component{
```

Comment:
This can also be a pure function.

Reply:
In other files you're using 
`class Something extends Component`
and here you're using 
`class FoodCard extends React.Component`

it's important to have consistency